mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactTableView.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ShowArtifactTableView from './ShowArtifactTableView';
+
+describe('ShowArtifactImageView', () => {
+  let wrapper;
+  let minimalProps;
+  let commonProps;
+
+  beforeEach(() => {
+    minimalProps = {
+      path: 'fakePath',
+      runUuid: 'fakeUuid',
+    };
+    commonProps = { ...minimalProps };
+    wrapper = shallow(<ShowArtifactTableView {...commonProps} />);
+  });
+
+  test('should render with minimal props without exploding', () => {
+    wrapper = shallow(<ShowArtifactTableView {...minimalProps} />);
+    expect(wrapper.length).toBe(1);
+  });
```

Comment:
Can we test the following behaviors?

1. Render CSV /TSV file fine.
2. When the number of rows is > 500, only show the first 500 rows.
3. Render the raw text if it's not valid CSV.

Reply:
I'm having some issues handling test case 2, can you please review the last test in `ShowArtifactTableView.test.js` in the latest commit and help me understand why `wrapper` is an empty array?

Reply:
`mount` needs to be replaced with `mountWithIntl` since `ShowArtifactTableView` contains `FormattedMessage`.